.github/workflows/dashboard-deploy.yml

```
@@ -0,0 +1,226 @@
+  notify-deployment:
+    name: Notify Dashboard Deployment
+    runs-on: ubuntu-latest
+    needs: [generate-data, deploy]
+    if: always()
+    steps:
+      - name: Notify Slack
+        if: success()
+        uses: 8398a7/action-slack@v3
+        with:
+          status: success
+          text: |
```

actionlint (1.7.8)

property "page_url" is not defined in object type {}

(expression)